scripts/base/frameworks/broker/backpressure.zeek

```diff
@@ -0,0 +1,35 @@
+##! This handles Broker peers that fall so far behind in handling messages that
+##! this node sends it that the local Broker endpoint decides to unpeer them.
+##! Zeek captures this as follows:
+##!
+##! - In broker.log, with a regular "peer-removed" entry indicating CAF's reason.
+##! - Via eventing through :zeek:see:`Broker::peer_removed` as done in this script.
+##!
+##! The cluster framework additionally captures the unpeering as follows:
+##!
+##! - In cluster.log, with a higher-level message indicating the node names involved.
+##! - Via telemetry, using a labeled counter.
+
+event Broker::peer_removed(endpoint: Broker::EndpointInfo, msg: string)
+	{
+	if ( "caf::sec::backpressure_overflow" !in msg ) {
+		return;
+	}
+
+	if ( ! endpoint?$network ) {
+		Reporter::error(fmt("Missing network info to re-peer with %s", endpoint$id));
+		return;
+	}
+
+	# Re-establish the peering so Broker's reconnect behavior kicks in once
+	# the other endpoint catches up. Broker will periodically re-try
+	# connecting as necessary. If the other endpoint originally connected to
+	# us, our attempt will fail (since we attempt to connect to the peer's
+	# ephemeral port), but in that case the peer will reconnect with us once
+	# it recovers.
```

Comment:
Is it correct that a node that disconnected a "connecting node" will endlessly attempt to connect to the other node's stale ephemeral port though and produce connection errors continuously?

Reply:
I see the comment below and don't have a better idea, unless `endpoint` could somehow be enriched with information whether the peer was originally "accepted" or "peered_to".

If the `cluster/setup-connections.zeek` / `Broker::peer()` functionality was idempotent, maybe could just re-run that, but don't think this works just like this.

So, don't know what to do.

Reply:
> All in all I feel I can live with this for now, assuming the assumption that backpressure problems are rare holds. But I'm happy to look into this in a follow-up.

The symptom for users would be "spurious" connection attempts on nodes to addr/ports that aren't anywhere in the cluster layout. This might be rather confusing, but follow-up / past 7.1 :+1: